### ISSUE TITLE
Add PIP to Linux Images

### ISSUE THREAD

**Issue by jonbgallant** [Microsoft/vsts-image-generation#72]

**Comment by jonbgallant** [comment]
@bryanmacfarlane

**Comment by bryanmacfarlane** [comment]
@DavidStaheli - can someone pick this up?

**Comment by DavidStaheli** [comment]
@brcrista did you already complete this?   FYI @madhurig.

**Comment by brcrista** [comment]
Yes

**Comment by bryanmacfarlane** [comment]
@brcrista - just confirming that we did on the Linux image as well.  I think so - just making sure.

**Comment by brcrista** [comment]
Yes, it's on the Linux image.  @stepro had actually added a system-wide Python and Pip a couple of weeks before we did the side-by-side versions in the tool cache.  But the tool cache versions have Pip as well, and those get put in PATH when you run the "Use Python Version" task

**Comment by bryanmacfarlane** [comment]
Great.  Thanks for confirming.

I think a default version of python and pip should be in the path and then folks can use the "use python" task to switch versions if that's not what they want.  That aligns with the hosted model as well where latest is available, use installers to switch.

If that's done, great.  If not, it would be very straight forward to pre-pend the path in the gen script pointing to a specific one in the cache.

**Comment by bryanmacfarlane** [comment]
With Linux pool coming out of preview, we’re creating a new image.  @bcrista, chat with David in my team and we can consolidate

**Comment by brcrista** [comment]
Ok.  The default version is in PATH.  I'll get in touch with David and see if he wants to consolidate.
